packages/renderer/src/lib/kube/pods/PodEmptyScreen.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
+import PodIcon from '../../images/PodIcon.svelte';
+import KubernetesEmptyScreen from '../KubernetesEmptyScreen.svelte';
```

Comment:
nit: `/@..` imports

Reply:
I don't love that this is different from all 7 other Kubernetes empty pages 🙈, but fixed.

Reply:
I have not looked on how to enforce that rule of using prefix scoping rather than relative scope